Fri May  6 14:44:27 EEST 2011  Sergei Trofimovich <slyfox@community.haskell.org>
  * follow ImplicitFromScope* -> Instance* renaming
  
  Fixes the following build failure
  
  [198 of 213] Compiling Agda.TypeChecking.Rules.Decl ( src/full/Agda/TypeChecking/Rules/Decl.hs, dist/build/Agda/TypeChecking/Rules/Decl.o )
  
  src/full/Agda/TypeChecking/Rules/Decl.hs:303:26:
      Not in scope: data constructor `ImplicitFromScope'
diff -rN -u old-Agda/src/full/Agda/TypeChecking/Rules/Decl.hs new-Agda/src/full/Agda/TypeChecking/Rules/Decl.hs
--- old-Agda/src/full/Agda/TypeChecking/Rules/Decl.hs	2011-05-06 14:54:17.920713001 +0300
+++ new-Agda/src/full/Agda/TypeChecking/Rules/Decl.hs	2011-05-06 14:54:18.450713001 +0300
@@ -300,7 +300,7 @@
               | (i, Arg h r _) <- zip [0..] $ reverse $ telToList tel
               ]
       telh' h	 = telFromList $ htel ++ [Arg h Relevant ("r", rect)]
-      telIFS	 = telh' ImplicitFromScope
+      telIFS	 = telh' Instance
 
   addCtxTel telIFS $ do
     allArgs <- getContextArgs
